### PR TITLE
Show pkg name and url when installing istio on VM

### PR DESCRIPTION
**Pull request by chxchx**
fixes https://github.com/istio/istio/issues/3600

### CODE REVIEW

**Comment by chxchx** [comment]
/assign @ldemailly 
The test you referred to in the issue seems stable on Prow now. I added more information in the VM installation script so users know what is being downloaded from where.

**Comment by ldemailly** [comment]
but how come it didn't download it in that prow job ?

File: install/tools/setupIstioVM.sh [hunk 1]
```diff
@@ -56,6 +56,7 @@ function istioNetworkInit() {
 # will generate and copy the files and install the packages on each machine.
 function istioInstall() {
   echo "*** Fetching istio packages..."
+  echo "Downloanding istio-sidecar.deb from {PILOT_DEBIAN_URL}/istio-sidecar.deb"
   # Current URL for the debian files artifacts. Will be replaced by a proper apt repo.
   rm -f istio-sidecar.deb
   echo curl -f -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > ${ISTIO_STAGING}/istio-sidecar.deb
```
Comment (ldemailly) [inline install/tools/setupIstioVM.sh]: the bug (from #2820) is here

please add the missing "" in that echo and you don't need the first one

Comment (chxchx) [reply install/tools/setupIstioVM.sh]: Done

File: install/tools/setupIstioVM.sh [hunk 2]
```diff
@@ -58,7 +58,7 @@ function istioInstall() {
   echo "*** Fetching istio packages..."
   # Current URL for the debian files artifacts. Will be replaced by a proper apt repo.
   rm -f istio-sidecar.deb
-  echo curl -f -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > ${ISTIO_STAGING}/istio-sidecar.deb
+  echo "curl -f -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > ${ISTIO_STAGING}/istio-sidecar.deb"
```
Comment (ldemailly) [inline install/tools/setupIstioVM.sh]: maybe "Running: curl ..." 
but it's fine as is too

**Pull request merged**